src/capitalsFirst.js

```diff
@@ -17,8 +17,30 @@
  *
  * @returns {string} - string with uppercase words in front
  */
+
+function isUpperCase(firstLetter) {
+  return (firstLetter >= 'A' && firstLetter <= 'Z');
+}
+
+function isLowerCase(firstLetter) {
+  return (firstLetter >= 'a' && firstLetter <= 'z');
+}
+
 function capitalsFirst(str) {
-  // write code here
+  let lowerCaseWords = [];
+  let upperCaseWords = [];
+  let inputWords = str.split(' ');
+  let firstLetter = '';
+
+  inputWords.forEach(function(item, i, arr) {
```

Comment:
use [arrow functions](https://mate-academy.github.io/style-guides/javascript.html#arrows--use-them) in callbacks